src/engine/client/ClientApplication.cpp

```diff
@@ -48,9 +48,11 @@ class ClientApplication : public Application {
             #endif
             #ifdef BUILD_GRAPHICAL_CLIENT
                 traits.isClient = true;
+                traits.supportsUri = true;
```

Comment:
Since it's true for all clients, it doesn't need to go in an ifdef.

Reply:
Moved out of ifdef.